src/renderer/routes/test_sequencer_panel/utils/SequenceHandler.ts

```diff
@@ -162,78 +200,87 @@ function buildResultFromCatch(e: unknown): Result<null, Error> {
   if (e instanceof Error) {
     return Err(e);
   } else {
-    console.error("[SaveProject] Unknown error", e);
-    return Err(new Error("Unknown error while creating the project"));
+    console.error("[Save Sequence] Unknown error", e);
+    return Err(new Error("Unknown error while creating the sequence."));
   }
 }
```

Comment:
Huuummm, each exposed fonction would be:
```
export function createSequence(...): Result<...> {
    return fromThrowable(() -> {
       ... actual function
     })
```
Instead of a try catch, kinda ugly

Reply:
(I'm not a fan of arrow function everywhere ahah)

Reply:
actually if we're not throwing anything anywhere we don't need to use `fromThrowable` at all, either just return errors early or chain together `andThen`

Reply:
Done :)